projects/ion/src/lib/icon/icon.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { render } from '@testing-library/angular';
+import { IonIconComponent, IonIconProps } from './icon.component';
+
+const sut = async (customProps?: IonIconProps): Promise<HTMLElement> => {
+  let type = 'trash';
+
+  await render(IonIconComponent, {
+    componentProperties: customProps || {
+      type: 'trash',
+    },
+  });
+  customProps && (type = customProps.type);
+  return document.getElementById('ion-icon-' + type);
+};
+
+describe('IonIconComponent', () => {
+  it('should render a correct icon', async () => {
+    await sut({ type: 'pencil' });
+    const elementRendered = document.getElementById('ion-icon-pencil');
+    expect(elementRendered).toBeTruthy();
+  });
+
+  it('should render icon with default size', async () => {
+    const svgElement = await sut();
+    const defaultSize = '24';
+    expect(svgElement.getAttribute('height')).toBe(defaultSize);
+    expect(svgElement.getAttribute('width')).toBe(defaultSize);
+  });
+
+  it('should render icon with default color', async () => {
+    expect((await sut()).getAttribute('fill')).toBe('#282b33');
+  });
```

Comment:
```suggestion
  it('should render icon with default color', async () => {
    const defaultColor = '#282b33';
    expect((await sut()).getAttribute('fill')).toBe(defaultColor);
  });
```
Since you're declaring defaultSize above and below, I think you can make this a pattern to this constant too.